repour/info.py

```diff
@@ -0,0 +1,20 @@
+import logging
+from aiohttp import web
+
+logger = logging.getLogger(__name__)
+
+#
+# Info/Documentation endpoint
+#
+
+# @asyncio.coroutine
+class InfoEndpoint:
```

Comment:
Not a fan of this being a class (I generally avoid using them where inheritance isn't required), and the cache being handled through a class attribute instead of a object attribute.

I think a closure can be used to serve this equally well.